loki/transformations/single_column/annotate.py

```diff
@@ -263,8 +270,8 @@ def device_alloc_column_locals(cls, routine, column_locals):
 
         if column_locals:
             vnames = ', '.join(v.name for v in column_locals)
-            pragma = ir.Pragma(keyword='acc', content=f'enter data create({vnames})')
-            pragma_post = ir.Pragma(keyword='acc', content=f'exit data delete({vnames})')
+            pragma = ir.Pragma(keyword='loki', content=f'unscoped-data create({vnames})')
+            pragma_post = ir.Pragma(keyword='loki', content=f'unscoped-enddata delete({vnames})')
```

Comment:
This entire section is not covered by tests. What situation is this for, what would be required to capture this?
(Cc @mlange05)